deepmd/utils/tabulate.py

```diff
@@ -44,6 +45,15 @@ def __init__(self,
         self.exclude_types = exclude_types
         if self.type_one_side and len(self.exclude_types) != 0:
             raise RunTimeError('"type_one_side" is not compatible with "exclude_types"')
+        
+        # functype
+        if activation_fn.__name__ == 'tf.nn.tanh' or activation_fn.__name__ == 'tanh':
```

Comment:
I think you can directly check if the method is another method instead of checking its name
```suggestion
        if activation_fn is tf.nn.tanh:
```

Reply:
OK, I will have a try. Thank you.